OktaAuth.podspec

```diff
@@ -13,8 +13,6 @@ Integrate your native app with Okta using the AppAuth library.
 
   s.ios.deployment_target = '9.0'
   s.source_files = 'Okta/**/*'
-  s.dependency 'AppAuth', '~> 0.91.0'
-  s.dependency 'Vinculum', '~> 0.2.0'
-  s.dependency 'HydraAsync', '~> 1.0.1'
-  s.dependency 'OktaJWT', '~> 1.0.0'
+  s.dependency 'AppAuth', '~> 0.95.0'
+  s.dependency 'HydraAsync', '~> 1.2.1'
```

Comment:
As a follow up, I'd love to find a way to remove `HydraAsync` as one of our dependencies. Fairly large task, but we should explore it in the future.

Reply:
Let's not handle this right now.

Reply:
Agree, but it is deeply integrated. Let's create an issue and prioritize it.

Reply:
@allebedew Can you create an issue to track that?

Reply:
Created https://github.com/okta/okta-sdk-appauth-ios/issues/86